Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,40 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string firstName;
+            string lastName;
+
+            int age;
+            double weight;
+            double height;
+
+            double bmi;
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("What is your First Name?");
+                firstName = Console.ReadLine();
+
+                Console.WriteLine("\nWhat is your Last Name?");
```

Comment:
Might be slightly better for readability to end strings with \n instead of beginning.
Also keep in mind Console.WriteLine(); creates a new line as method name implies.
Console.Write(); writes to console without starting on new line.
Just being picky, overall good work!